src/main/java/Game.java

```diff
@@ -45,8 +53,8 @@ private Player getWinner() {
         return currentPlayer == player1 ? player2 : player1;
     }
 
-    private boolean winFor(String symbol, int position) {
-        List<String> win = Arrays.asList(symbol, symbol, symbol);
+    private boolean winFor(String mark, int position) {
+        List<String> win = Arrays.asList(mark, mark, mark);
         return board.winningPositions().get(position).equals(win);
     }
```

Comment:
There is a lot of method chaining on line 58. Could you ask the board if the position is a winning position, rather than the game figuring it out? Eg: board.isWin(position) This would hide all the chained calls in board.